lib/vm/src/memory.rs

```diff
@@ -555,4 +613,7 @@ where
 
         initialize_memory_with_data(memory, start, data)
     }
+
+    /// Copies this memory to a new memory
+    fn fork(&mut self) -> Result<Box<dyn LinearMemory + 'static>, MemoryError>;
```

Comment:
Let's use clone. I think is a more accurate naming

Reply:
Renamed to `duplicate()`.